### PR TITLE
Fix planning tool auto-selection

### PR DESCRIPTION
**Pull request by moshthepitt**
This PR addresses some issues on the planning tool:

- Ensures that the planning tool can work even when the API is broken:
    - it works even when the jurisdiction hierarchy API does not return a `structureCount` property
    - it now works even when the jurisdiction metadata API does not return valid `value` properties
- Adds information to the risk slider to show the number of selected jurisdictions.  I think that this is necessary so that the
  user can see how many jurisdictions are selected.  It was possible in the past to move the slider and have the structure count
  not change, but the jurisdiction count would be changing.
- Use `>=` instead of `>` when evaluating if a jurisdiction meets criteria for selection based on risk.  This would cater for jurisdictions whose values are 0 or 100, which would otherwise not always be selected when they should. 

Fix: #1442 
Part of fixing: #1441

add-jurisdiction-count

### CODE REVIEW

File: src/store/ducks/opensrp/hierarchies/index.tsx [hunk 1]
```diff
@@ -541,7 +541,7 @@ export const getStructuresCount = () =>
     const allSelectedLeafNodes = computeSelectedNodes(tree, true, metaByJurisdiction);
     const reducingFn = (acc: number, value: number) => acc + value;
     return allSelectedLeafNodes
-      .map(node => node.model.node.attributes.structureCount)
+      .map(node => node.model.node.attributes.structureCount || 0)
```
Comment (moshthepitt) [inline src/store/ducks/opensrp/hierarchies/index.tsx:544]: Ensure that we don't fail when `structureCount` is not defined.

File: src/containers/pages/JurisdictionAssignment/helpers/Slider/index.tsx [hunk 2]
```diff
@@ -131,6 +136,13 @@ export const JurisdictionSelectionsSlider = (props: Props) => {
               <p>{structuresCount}</p>
             </Col>
           </Row>
+          <Row className="auto-target-row mb-3">
+            <Col xs="12" md="12" className="info-section py-0 px-5">
+              <p className="text-center mt-3">
+                {jurisdictionsCount} {JURISDICTIONS_SELECTED}
+              </p>
+            </Col>
+          </Row>
```
Comment (moshthepitt) [inline src/containers/pages/JurisdictionAssignment/helpers/Slider/index.tsx:145]: Added a way to display to the user how many jurisdictions are selected.  See commit message: https://github.com/onaio/reveal-frontend/pull/1443/commits/73d347ad35de49bc80f38f6511d046641f28fd03

File: src/containers/pages/JurisdictionAssignment/helpers/Slider/tests/slider.test.tsx [hunk 3]
```diff
@@ -39,31 +67,111 @@ describe('JurisdictionAssignment/Slider', () => {
...
+    // simulate another change to deselect all jurisdictions
+    /** we really should not be doing this, but I am currently unable to simulate
+     * the change event
+     */
+    act(() => {
+      (wrapper.find('InputRange').props() as any).onChange(81);
+    });
+    expect(wrapper.find('.risk-label').text()).toMatchInlineSnapshot(`"81%"`);
     (wrapper.find('InputRange').props() as any).onChangeComplete(81);
+    // we should now have two jurisdictions selected
```
Comment (p-netm) [inline src/containers/pages/JurisdictionAssignment/helpers/Slider/tests/slider.test.tsx:125]: I think this should read `no selected jurisdictions`

**Review by p-netm — COMMENTED** [review]
:+1:

File: src/containers/pages/JurisdictionAssignment/helpers/Slider/tests/slider.test.tsx (hunk 3 above)
Comment (moshthepitt) [reply src/containers/pages/JurisdictionAssignment/helpers/Slider/tests/slider.test.tsx]: thanks for catching that @p-netm - now fixed

**Review by p-netm — APPROVED** [review]

**Review by ciremusyoka — APPROVED** [review]

**Pull request merged**